setup.py

```diff
@@ -61,13 +61,14 @@
 ]
 
 install_requires = [
+    'invenio-base>=1.2.4',
+    'Flask>=1.1.0,<2.0.0',
```

Comment:
this is "required" because the pip resolver takes ages to resolve the correct set of versions of all packages otherwise